### PR TITLE
Secure Signals test page for EUID

### PR DESCRIPTION
**Pull request by ssundahlTTD**

### CODE REVIEW

File: static/examples/cstg-secure-signals-example/index.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>UID2 Publisher Client-Side Integration Example using UID2 JavsScript SDK, Secure Signals</title>
```
Comment (genwhittTTD) [inline static/examples/cstg-secure-signals-example/index.html:5]: JavaScript

File: static/examples/cstg-secure-signals-example/index.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    <link rel="stylesheet" type="text/css" href="./styles/ads.css" />
+    <link rel="stylesheet" type="text/css" href="./styles/app.css" />
+    <link rel="shortcut icon" href="images/favicon.png" />
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script>
+
+    </script>
+  </head>
+  <body>
+    <h1>UID2 Publisher Client-Side Integration Example using UID2 JavsScript SDK, Secure Signals</h1>
```
Comment (genwhittTTD) [inline static/examples/cstg-secure-signals-example/index.html:15]: JavaScript

File: static/examples/cstg-secure-signals-example/index.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    <p>
+      This example demonstrates how a content publisher can follow the
+      <a href="https://unifiedid.com/docs/guides/publisher-client-side"
```
Comment (genwhittTTD) [inline static/examples/cstg-secure-signals-example/index.html:18]: Use updated URL: https://unifiedid.com/docs/guides/integration-javascript-client-side
(there is a redirect, but we should update to the new URL everywhere we have the old one)

File: static/examples/cstg-secure-signals-example/index.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+        >Client-Side Integration Guide for JavaScript
+      </a>
+      to implement UID2 integration and generate UID2 tokens.
+      Secure Signals is updated when the page is reloaded.  
+      Reload the page when logging in order out to update Secure Signals in local storage.
```
Comment (genwhittTTD) [inline static/examples/cstg-secure-signals-example/index.html:23]: "Reload the page when logging in order out to update..."
Not sure exactly what this should say. Maybe "in order to update"?
Not sure about the logging, prob fine? Or does it mean the user logging in? We've removed a lot of mentions of user login.

File: static/examples/cstg-secure-signals-example/scripts/ads.js [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+      adsManager.start();
+    } catch (adError) {
+      // Play the video without ads, if an error occurs
+      console.log("AdsManager could not be started");
+      videoElement.play();
+    }
+}
+
+function onAdsManagerLoaded(adsManagerLoadedEvent) {
+  // Instantiate the AdsManager from the adsLoader response and pass it the video element
```
Comment (genwhittTTD) [inline static/examples/cstg-secure-signals-example/scripts/ads.js:95]: add period at the end.

File: static/examples/cstg-secure-signals-example/scripts/scripts.js [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+        $('#login_form').show();
+        $('#logout_form').hide();
+    } else {
+        $('#login_form').hide();
+        $('#logout_form').show();
+    }
+
+    const secureSignalsStorage = localStorage['_GESPSK-euid.eu'];
+    if (token && !secureSignalsStorage) {
+        // token is valid but secure signals has not been refreshed.  reload the page
```
Comment (genwhittTTD) [inline static/examples/cstg-secure-signals-example/scripts/scripts.js:28]: caps for Secure Signals, period at the end.

// Token is valid but Secure Signals has not been refreshed. Reload the page.

**Review by genwhittTTD — CHANGES_REQUESTED** [review]
Great that we're adding this! Left a few copy mods.

File: static/examples/cstg-secure-signals-example/index.html (hunk 3 above)
Comment (ssundahlTTD) [reply static/examples/cstg-secure-signals-example/index.html:18]: is there an EUID version of this?  Or should the EUID docs point to unifiedid?

File: static/examples/cstg-secure-signals-example/index.html (hunk 4 above)
Comment (ssundahlTTD) [reply static/examples/cstg-secure-signals-example/index.html:23]: sorry.  changed to "Reload the page in order to update Secure Signals in local storage."

File: docs/guides/integration-google-ss.md [hunk 7]
```diff
@@ -107,6 +107,8 @@ This script is hosted on CDN, and GPT automatically loads it with the secure sig
 
 For details, see [Client-Side Integration Guide for JavaScript](integration-javascript-client-side.md).
 
+A sample application is also available for integration using the SDK for JavaScript.  See [Sample application](https://euid.eu/examples/cstg-secure-signals-example/).
```
Comment (ssundahlTTD) [inline docs/guides/integration-google-ss.md:110]: Added a link to the new sample page

File: static/examples/cstg-secure-signals-example/index.html [hunk 8]
```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>UID2 Publisher Client-Side Integration Example using UID2 JavaScript SDK, Secure Signals</title>
```
Comment (genwhittTTD) [inline static/examples/cstg-secure-signals-example/index.html:5]: Is it fine that we're saying UID2 here but we're on the EUID site? Didn't notice that before sorry. Lines 5, 15, 21. Just checking.

File: static/examples/cstg-secure-signals-example/index.html [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script>
+
+    </script>
+  </head>
+  <body>
+    <h1>UID2 Publisher Client-Side Integration Example using UID2 JavaScript SDK, Secure Signals</h1>
+    <p>
+      This example demonstrates how a content publisher can follow the
+      <a href="https://unifiedid.com/docs/guides/integration-javascript-client-side"
```
Comment (genwhittTTD) [inline static/examples/cstg-secure-signals-example/index.html:18]: Ah oops I missed this too. The link is to the UID2 doc, but this is EUID. The URL is:
https://euid.eu/docs/guides/integration-javascript-client-side
Doc title is the same.

**Review by genwhittTTD — APPROVED** [review]
Found a couple of other fixes needed. Otherwise looks great. Approving with changes. Thanks much!

File: static/examples/cstg-secure-signals-example/index.html (hunk 8 above)
Comment (ssundahlTTD) [reply static/examples/cstg-secure-signals-example/index.html:5]: changed to EUID

**Pull request merged**